presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2535,6 +2538,124 @@ public void testCreateTableUnsupportedType()
         }
     }
 
+    @Test
+    public void testCreateEmptyBucketedTemporaryTable()
+            throws Exception
+    {
+        List<ColumnMetadata> columns = ImmutableList.<ColumnMetadata>builder()
```

Comment:
hmm... I am wondering if it makes sense to put into a static final such as `CREATE_TEMPORARY_TABLE_COLUMNS`, similar to `CREATE_TABLE_COLUMNS` ? Similar to the data...

Reply:
Yeah, i extracted pretty much everything into constants, and merged 2 test methods into a single one that checks both, non empty and empty cases.